containerd-shim-spin/src/engine.rs

```diff
@@ -367,6 +376,25 @@ impl SpinEngine {
         }
         None
     }
+
+    async fn handle_archive_layer(
+        &self,
+        cache: &Cache,
+        bytes: impl AsRef<[u8]>,
+        digest: impl AsRef<str>,
+    ) -> Result<()> {
+        // spin_oci::client::unpack_archive_layer attempts to create a tempdir via tempfile::tempdir()
+        // which will fall back to /tmp if TMPDIR is not set. /tmp is either not found or not accessible
+        // in the shim environment, hence setting to current working directory.
+        if env::var("TMPDIR").is_err() {
+            log::debug!(
```

Comment:
When do we expect `TMPDIR` to be set? I am wondering if this log is necessary.

Reply:
Good point... it appears it isn't ever set, currently.  Glad to remove this debug log if you'd like.

Reply:
I think we would want to leave it, otherwise we might overwrite it. I don't know what scenario it would be set in, but if it was set and we overwrite I don't think that would be an expected behavior?